go/database/mpt/state.go

```diff
@@ -297,7 +300,9 @@ func (s *MptState) GetStorage(address common.Address, key common.Key) (value com
 func (s *MptState) SetStorage(address common.Address, key common.Key, value common.Value) error {
 	return s.trie.SetValue(address, key, value)
 }
-
+func (s *MptState) HasEmptyStorage(address common.Address) (bool, error) {
```

Comment:
please ad doc + add a simple test checking that the call pass through,

Reply:
I combined `live_trie` and `state_test` into one. Since it only checks for call this covers both. Wdyt?

Reply:
LiveTrie should have its test as well - a very simple one, checking the the method call passed through